lib/opsbot.js

```diff
@@ -84,45 +84,28 @@ module.exports.sendSlackAlmaMessage = (msg, channel, options, callback) => {
   });
 };
 
-module.exports.sendSlackAlmaMessageAsync = util.promisify((msg, channel, options, callback) =>
-  module.exports.sendSlackAlmaMessage(msg, (err, res, body) => callback(err, { res, body }))
```

Comment:
This would have never worked to begin with - it's missing two arguments (channel and options).

Reply:
Checked the git blame to see who put this in...

https://pbs.twimg.com/media/FEz4HC5XsAMztm3?format=jpg&name=large